### ISSUE TITLE
Moiré Pattern in FBP reconstruction 

### ISSUE THREAD

**Issue by oooo1114** [LLNL/LEAP#138]
Hi Kyle,
I tested FBP with my private head data, but found Moiré Pattern in the lower part near the bony area.
Do you have any ideas, is it related to the distance dirven backward projector?
Thanks.
![image](https://github.com/user-attachments/assets/04ff6939-301a-47c5-9f6b-a7bd51323382)

**Comment by kylechampley** [comment]
It's not direct;y related to our backprojection algorithm; if anything the backprojectors in LEAP should give smoother results than other backprojection methods.  It looks like you have a very tight display window and you might just be amplifying ringing artifacts that are inherent to FBP-type reconstructions.  Or maybe a mistake was made somewhere.

**Comment by oooo1114** [comment]
Yes, I am using a very tight window, like 0-90 HU for brain. But it is werid the pattern only appeared in the lower part, the middle part of barin looks fine. Do u have any more suggestions?
![image](https://github.com/user-attachments/assets/eff27fd1-8eeb-4dc3-8b20-9214a0f217f9)

**Comment by oooo1114** [comment]
Hi Kyle, I also play with the type of projector. In the following imge, the left one is with voxel-driven projector and the right one is with SF proector. It is strange the high frequency artifacts are stronger in the SF one. 
Do you have any ideas?
Thanks.
![image](https://github.com/user-attachments/assets/49cad01c-80a1-439d-b8f6-a1b064f807f4)

**Comment by kylechampley** [comment]
That's a bit strange.  Could you send me your script or at least the output of leapct.print_parameters()

**Comment by oooo1114** [comment]
Here are the output:
======== CT Cone-Beam Geometry (curved detector) ========
number of angles: 1440
number of detector elements (rows, cols): 32 x 860
angular range: 360.000000 degrees
detector pixel size: 1.120000 mm x 1.064500 mm
center detector pixel: 15.500000, 425.245087
sod = 570.000000 mm
sdd = 1005.000000 mm

======== CT Volume ========
number of voxels (x, y, z): 544 x 544 x 30
voxel size: 0.449219 mm x 0.449219 mm x 1.270448 mm
volume offset: 0.000000 mm, -19.299999 mm, 0.000000 mm
FOV: [-122.187500, 122.187500] x [-141.487503, 102.887497] x [-19.056715, 19.056715]
